### DISCUSSION TITLE
dispatchCommand can't run CustomAlias command is normal?

### DISCUSSION THREAD

**Discussion by alan67160** [Zrips/CMI#4946] [Q&A]
bukkit.getServer().dispatchCommand(Bukkit.getServer().getConsoleSender(),"SomeCustomAliasCommand")

**Comment by mrfloris** [comment]
If you still have this issue with the latest CMILIB and CMI jar files for 1.17.1, can you please clarify a bit what you're trying to do or when you're getting this?
